x-pack/platform/plugins/shared/alerting/common/routes/gaps/apis/get_rules_with_gaps/schemas/v1.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+
+export const getRuleIdsWithGapQuerySchema = schema.object(
+  {
+    end: schema.string(),
+    start: schema.string(),
+    statuses: schema.maybe(schema.arrayOf(schema.string())),
+  },
+  {
+    validate({ start, end }) {
+      if (start) {
```

Comment:
should we validate that start < end?

Reply:
https://github.com/elastic/kibana/commit/aa510df65bec67b01811160d05132a13c2793594